src/open_inwoner/templates/pages/ssd/monthly_reports_list.html

```diff
@@ -1,54 +1,41 @@
-                    {% if client.config.maandspecificatie_enabled is True %}
-                        <div class="tabs__body">
-                        {# Note: Mijn uitkeringen tab-content styles need to be independent from the URL, unlike the Tabs of the Login page #}
-                        {# URL-dependent styles would be coming from src/open_inwoner/js/components/tab-panels #}
-                        {# Setting class to 'tab__contents' instead of 'tab__content' is a quick-fix #}
-                            <div id="maandspecificaties" class="tab__contents active">
-                                <p>
-                                    {% blocktrans with display_text=client.config.maandspecificatie_display_text %}
-                                        {{ display_text }}
-                                    {% endblocktrans %}
-                                </p>
-                                {% render_form form=form method="POST" id="monthlyreports-form" show_required=False submit_text=_('Download') %}
-                                    {% csrf_token %}
-                                    {% input form.report_date icon="expand_more" icon_position="after" icon_outlined=True %}
-                                    {% form_actions primary_text=_("Download") primary_icon="file_download" icon_position="before" extra_classes="ssd" %}
-                                {% endrender_form %}
-                                <div>
-                                    {% if report_not_found %}
-                                        {% blocktrans with time_period=report_not_found|date:"F Y" %}
-                                            Geen uitkeringsspecificatie gevonden voor {{ time_period }}.
-                                        {% endblocktrans %}
-                                    {% endif %}
-                                </div>
-                            </div>
+{% block report %}
+    <div class="tab--container">
+        <div class="tabs">
+            <ul class="list tabs__headers">
+                <li class="list-item tab__header--item"><span id="monthly" class="link tab__header active">{% trans "Maandspecificaties" %}</span></li>
+                <li class="list-item tab__header--item"><a href="{% url 'ssd:yearly_benefits_index' %}" id="yearly" class="link tab__header">{% trans "Jaaropgaven" %}</a></li>
+            </ul>
+            {% if client.config.maandspecificatie_enabled is True %}
+                <div class="tabs__body">
+                {# Note: Mijn uitkeringen tab-content styles need to be independent from the URL, unlike the Tabs of the Login page #}
+                {# URL-dependent styles would be coming from src/open_inwoner/js/components/tab-panels #}
+                {# Setting class to 'tab__contents' instead of 'tab__content' is a quick-fix #}
+                    <div id="maandspecificaties" class="tab__contents active">
+                        <div class="wysiwyg">
```

Comment:
Very nice :-)

But now that we removed styling for the paragraph, we still need some spacing for the tab-panel, so I'd suggest to go to `src/open_inwoner/scss/components/TabPanel/TabPanel.scss` and then add the 'quickfix' tab__contents classes to the existing styles (these will change in another PR that changes the styling for Uitkeringen only and will make it different, but still look like the Login page Tabs anyway later on):

```
  .tab__content,
  .tab__content.active,
  .tab__contents,
  .tab__contents.active {
    margin: 1em;
    scroll-margin-top: 150px;
  }
```

TLDR: add the classes that have an extra 's' in the end of their name.